client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,27 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
```

Comment:
I totally agrre with you, but its a temporary data will be replaced into https://github.com/cozy/cozy-proxy/pull/310, see https://github.com/cozy/cozy-proxy/pull/310/files#diff-6487018685f0002494bb58611dd1d9fcR36